include/deal.II/dofs/dof_accessor.h

```diff
@@ -1588,6 +1588,9 @@ class DoFCellAccessor :  public DoFAccessor<DH::dimension,DH, level_dof_access>
    * @{
    */
 
+  // Necessary for Visual Studio for not to replace the get_fe version with one parameter with the following parameter-less version.
+  using DoFAccessor<DH::dimension, DH, level_dof_access>::get_fe;
+
```

Comment:
Error that get_fe does not take 1 argument (because the compiler sees only the parameter-less version), because of name-hiding. More e.g. here:

http://stackoverflow.com/questions/1628768/why-does-an-overridden-function-in-the-derived-class-hide-other-overloads-of-the